app/next-client-app/components/core/UserMenu.tsx

```diff
@@ -38,15 +39,17 @@ export async function UserMenu({ username }: { username?: string }) {
         <DropdownMenuLabel>My Account</DropdownMenuLabel>
         <DropdownMenuSeparator />
         {/* TODO: Add change password here */}
-        {/* <DropdownMenuGroup>
-          <DropdownMenuItem>
-            <Settings className="icon-md mr-2" />
-            <span>Change Password</span>
-            <DropdownMenuShortcut>⌘S</DropdownMenuShortcut>
-          </DropdownMenuItem>
+        <DropdownMenuGroup>
+          <Link href="/password-reset" passHref>
+            <DropdownMenuItem asChild>
+              <button className="flex items-center w-full">
+                <Settings className="icon-md mr-2" />
+                <span>Change Password</span>
```

Comment:
```suggestion
                <span>Reset Password</span>
```
![Screenshot 2025-04-17 at 11 38 31](https://github.com/user-attachments/assets/284627ce-09e4-496b-8ba2-6814747ca8f8)